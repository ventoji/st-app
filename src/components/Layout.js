import React from 'react';

const Layout = ({ className = 'container', children }) => (
  <div className={className}>{children}</div>
);

export default Layout;
