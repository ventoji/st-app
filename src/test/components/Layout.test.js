import React from 'react';
import Layout from '../../components/Layout';
import { toMatchSnapshotFunction } from '../fixture/helper';

test('should render Layout coorectly', () => {
  toMatchSnapshotFunction(<Layout />);
});
