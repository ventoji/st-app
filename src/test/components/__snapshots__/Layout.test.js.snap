// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header coorectly 1`] = `
<div
  className="container"
/>
`;

exports[`should render Layout coorectly 1`] = `
<div
  className="container"
/>
`;
